expression/util.go

```diff
@@ -789,6 +789,14 @@ func GetUint64FromConstant(expr Expression) (uint64, bool, bool) {
 		return uint64(val), false, true
 	case types.KindUint64:
 		return dt.GetUint64(), false, true
+	case types.KindString:
+		valStr := dt.GetString()
```

Comment:
We cannot add this check, because as I mentioned in the issue that in MySQL:
```
mysql> create table t(a int, b int);
Query OK, 0 rows affected (0.01 sec)

mysql> insert into t values(1,2),(1,3),(2,3),(-1,1),(-1,-1);
Query OK, 5 rows affected (0.01 sec)
Records: 5  Duplicates: 0  Warnings: 0

mysql> prepare stmt from "select sum(b) over w, nth_value(b, ?) over w from t window w as (partition by a)";
Query OK, 0 rows affected (0.00 sec)

mysql> set @a='1';
Query OK, 0 rows affected (0.01 sec)

mysql> execute stmt using @a;
ERROR 1210 (HY000): Incorrect arguments to nth_value
```

Reply:
Yeah, I got it.